elasticsearch/src/test/java/akka/stream/alpakka/elasticsearch/ElasticsearchTest.java

```diff
@@ -40,11 +40,14 @@
   @BeforeClass
   public static void setup() throws IOException {
     runner = new ElasticsearchClusterRunner();
-    runner.build(ElasticsearchClusterRunner.newConfigs().baseHttpPort(9210).baseTransportPort(9310).numOfNode(1));
+    runner.build(ElasticsearchClusterRunner.newConfigs()
+        .baseHttpPort(9200)
+        .baseTransportPort(9300)
+        .numOfNode(1));
     runner.ensureYellow();
 
     //#init-client
-    client = RestClient.builder(new HttpHost("localhost", 9211)).build();
+    client = RestClient.builder(new HttpHost("localhost", 9201)).build();
```

Comment:
When (intentionally) re-using the same port, you could consider putting the actual port number in a constant somewhere and referring to it from both places where you use it.

Reply:
I think sharing the port number is not necessary. These tests are independent and they don't have to use same port necessarily. It's just no longer necessary to use different ports.

Reply:
Agreed, using a constant make it more obvious that it's intentional though. But OK like this :+1: